docker-compose.yml

```diff
@@ -12,7 +12,6 @@ services:
       - core_data:/dash
     command:
```

Comment:
Let's just remove the command?

Reply:
This is necessary to work around the BLS key in the dash.conf file when wallet functions are used in core.